shell/content/docs/en-us/whats-new.md

```diff
@@ -1,7 +1,26 @@
+---
+title: What's New in 2.7
+---
+
+## New in 2.7.0
+
+### Rancher Extensions
+
+- You can now bring your own apps' UI and integrate it with the Rancher Dashboard to leverage a single UI. This relies on feature charts that would be manageable via a catalog. 
+
 ---
 title: What's New in 2.6
 ---
 
+## New in 2.6.9
+
+- Alerts within the UI regarding network connectivity are now off by default with a toggle to enable them. This is to address usability uncovered as part of disconnected web sockets.
```

Comment:
The second sentence is a bit confusing to me, I would say more like: 
```suggestion
- Alerts within the UI regarding network connectivity are now off by default with a toggle to enable them. This will address usability concerns when web sockets are disconnected.
```